core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -158,6 +161,92 @@ public void testInitialize() {
     jdbcCatalog.initialize("test_jdbc_catalog", properties);
   }
 
+  @Test
+  public void testSchemaUpdate() throws Exception {
```

Comment:
minor: `testSchemaUpdate` isn't really telling much about what's exactly being tested, that's why I suggested to name this `schemaIsMigratedToAddViewSupport` so that it's explicit that we care about view support being added

Reply:
Yes, good point. I do the change.